src/runtime/HalideBuffer.h

```diff
@@ -75,12 +75,20 @@ struct AllocationHeader {
     std::atomic<int> ref_count {0};
 };
 
+/** An enumeartion indicating how to deallocate the device state for a
```

Comment:
Pushed a commit to fix this comment, but it doesn't seem to be showing up in the PR. If eventual consistency doesn't prevail, I'll poke at it.